src/templates/filtersDrawerNestedTogglesLabel.jsx

```diff
@@ -0,0 +1,59 @@
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../elements/icon';
+
+const propTypes = {
+    nestedTogglesData: PropTypes.shape({
+        label: PropTypes.string,
+    }).isRequired,
+    onClick: PropTypes.func.isRequired,
+};
+
+class FiltersDrawerNestedTogglesLabel extends React.PureComponent {
+    constructor() {
+        super();
+
+        this.onClick = this.onClick.bind(this);
+        this.onKeyDown = this.onKeyDown.bind(this);
+    }
+
+    onClick() {
+        const { nestedTogglesData, onClick } = this.props;
+
+        onClick(nestedTogglesData);
+    }
+
+    onKeyDown(event) {
+        const { nestedTogglesData, onClick } = this.props;
+
+        if (event.keyCode === 13) {
```

Comment:
Don't we have a `const` for this somewhere?

Reply:
Not yet. Do we need a `const`?

Reply:
I see this littered all over HC:
```JavaScript
const ENTER_KEY_CODE = 13;

class FooComponent extends React.PureComponent {

// ... blah blah blah ...

    onKeyDown(event) {
        if (event.keyCode === ENTER_KEY_CODE) {
            // do whatever it is supposed to do when 'Enter' was hit...
        }
    }

    // ... more blah blah blah ...
}
```

At some point we had discussed moving `const ENTER_KEY_CODE = 13;` into some shared constants file, exporting it from there and importing it everywhere it is needed, but apparently we never pulled the trigger on that.  Is it worth doing something like that in both projects (React-CM-UI and HC), or is that overkill?

Reply:
Yeah, you're right, I think it would be better to have a util `function` to handle the problem instead of a util `const`.  What do you think?

Reply:
something like this?
`utils/keyEventUtils.js`
```JavaScript
const ENTER_KEY_CODE = 13;

// ...

export function isEnterKeyEvent(event) {
    return _.isObject(event) && event.keyCode === ENTER_KEY_CODE;
}
```